.github/sync-repo-settings.yaml

```diff
@@ -7,8 +7,19 @@ branchProtectionRules:
   requiresCodeOwnerReviews: true
   requiresStrictStatusChecks: true
   requiredStatusCheckContexts:
-    - 'Kokoro'
-    - 'cla/google'
+    - 'cla/google',
+    - 'OwlBot Post Processor',
+    - 'docs',
+    - 'docfx',
+    - 'lint',
+    - 'unit (3.6)',
```

Comment:
Do we want to have consistency on 3.6 between unit tests and sample tests? I don't see 3.6 listed under samples.

Reply:
@kweinmeister We (Python samples folks) have mostly dropped Python 3.6 from the samples testing matrix as it went EOL in December 2021. It remains in the library unit test matrix since the library states it is compatible in the `setup.py`.  

Historically we've stopped testing samples against a particular version a bit earlier than we stop testing the library.